src/components/atoms/Announcement/Announcement.scss

```diff
@@ -0,0 +1,10 @@
+@import 'colors';
+
+.Announcement {
+  width: 100%;
+  margin-bottom: -1.5rem;
```

Comment:
Can you avoid negative margins?

Reply:
@wzalazar OK made the changes. Thank you!